apps/dashboard/src/components/workflow-editor/steps/component-utils.tsx

```diff
@@ -23,6 +25,12 @@ export const getComponentByType = ({ component }: { component?: UiComponentEnum
     case UiComponentEnum.URL_TEXT_BOX: {
       return <InAppRedirect />;
     }
+    case UiComponentEnum.MAILY: {
```

Comment:
@desiprisg consider aligning the naming on this one. All other components are based on the step type field, not on the implementation of the built-in ui or the underlying library. 

```suggestion
    case UiComponentEnum.EMAIL_BODY: {
```